Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/PluginManager.cpp

```diff
@@ -234,7 +233,6 @@ namespace EMStudio
         RegisterPlugin(new MotionSetsWindowPlugin());
         RegisterPlugin(new NodeGroupsPlugin());
         RegisterPlugin(new AnimGraphPlugin());
-        RegisterPlugin(new OpenGLRenderPlugin());
```

Comment:
One thing worth to hand checking is what happen when loading a old layout still with OpenGLRenderPlugin. It should ignore the plugin with some kind of warning msg saying something like "openGLPlugin is disabled, switch to atomRenderPlugin instead.

For this PR i think the current stage is ok - as long as loading the old layout won't introduce a crash.